### ISSUE TITLE
Realm database is empty

### ISSUE THREAD

**Issue by Delphian2015** [uPhyca/stetho-realm#45]
Hello.

Thank you for your plugin. I am trying to use it. I see my database but the base is empty although I have a data (4 realm object classes and data in them)?  
![chrome](https://cloud.githubusercontent.com/assets/14300153/21353569/acccb96e-c6d7-11e6-8c98-e0d15833d0e5.png)

**Comment by Farasy** [comment]
Faced the same issue.
Don't setup a custom name for the realm or custom folder solved this.

**Comment by careful7j** [comment]
Thank you sir, removing line ".name("myrealm2.realm") solved my problem, but It's quite sad that we can't use custom realm database names. I.e. I need more than one db than it can become an issue.
```
new RealmConfiguration.Builder()
                .name("myrealm2.realm")  //Comment this line so database won't be empty anymore
                .schemaVersion(2) 
                .migration(DbUtils.migration)
                .build();
```

**Comment by Delphian2015** [comment]
Yes, I deleted the name of the database, but it is not a good solution. Thank you!

**Comment by CYRUS-STUDIO** [comment]
@Delphian2015 I'm fixed the problem, you can see [this](https://github.com/uPhyca/stetho-realm/pull/46).

**Comment by marafat** [comment]
Any plans on releasing this fix soon??

**Comment by CYRUS-STUDIO** [comment]
stetho-realm 2.0.1 is released.

edit build.gradle:
```
repositories {
    maven { url 'https://github.com/linchaolong/stetho-realm/raw/master/maven-repo' }
}

dependencies {
    compile 'com.facebook.stetho:stetho:1.4.2'
    compile 'com.uphyca:stetho_realm:2.0.1'
}
```

**Comment by zaki50** [comment]
I've released 2.1.0 that contains the fix.
I'll close this issue.
